src/lib/utils/client-route.js

```diff
@@ -8,17 +8,18 @@ exports.register = (server, options, next) =>  {
         path: '/client/{param*}',
         method: 'GET',
         config: {
-            tags: ['client'],
-            notes: 'Add filename (e.g. client.js)',
-            description: 'Returns client source code',
             auth: false,
-            plugins: { 'hapi-swagger': { nickname: 'get' } },
+            cors: true,
```

Comment:
this is the only real change in this file.  otherwise, just alphabetizing.  srry :/